.travis.yml

```diff
@@ -32,31 +35,20 @@ before_install:
 
 install:
  - travis_retry cabal update
- # can't use "cabal install --only-dependencies --enable-tests" due to dep-cycle
- - cabal install
-      "QuickCheck >=2.10 && <2.14"
-      "byteorder ==1.0.*"
-      "dlist >=0.5 && <0.9"
-      "mtl >=2.0 && <2.3"
-      deepseq test-framework-hunit test-framework-quickcheck2
 
 script:
- - cabal configure --enable-tests -v2
- - cabal build
- # --show-details=streaming is available for CABALVER>=1.20 only
- - if [ "$(echo -e "1.20\n$CABALVER" | sort -rV | head -n1)" = "$CABALVER" ]; then
-     cabal test --show-details=streaming;
-   else
-     travis_wait cabal test --show-details=always;
-   fi
- - cabal sdist
+ - cabal v2-configure; cabal v2-build
+ - cabal v2-sdist -o .
+ # Remove to enable build of sdist.
+ - rm -f cabal.project
+ - rm -f tests/test-builder.tix
```

Comment:
This should be redundant after the removal of the `-fhpc` flag?

```suggestion
```

Reply:
Yes, but also harmless, if the -fhpc comes back, but if you'd like to tidy it up, I can remove it...

Reply:
I'd slightly prefer to have this removed.